docs/make.jl

```diff
@@ -28,6 +39,14 @@ pages = Any[
     "Arrays" => "Arrays.md",
     "DGmethods_old" => "DGmethods_old.md",
     "InputOutput.md",
+    "Examples" => [
+        "Conjugate Gradient" => "generated/example_cg.md",
+        "Literate" => "generated/literate_markdown.md",
+    ],
+    "Contribution Guides" => Any[
+        "ContributionGuides/CONTRIBUTING.md",
+        "ContributionGuides/IterativeSolvers.md",
```

Comment:
Should this go under developer docs?

Reply:
Yeah, I wasn't sure what made more sense, if you are leaning towards developer docs, then let's put it there.